docs/guides/admin/docs/configuration/database.md

```diff
@@ -103,10 +103,10 @@ Finally, leave the client and restart the database server to enable the new user
 % systemctl restart mariadb.service
 ```
 
-PostgreSQL
-----------
+Step 3: PostgreSQL
```

Comment:
Why is MariaDB step 2 and PostgreSQL step 3? You only need one of them but having them as subsequent steps suggests that you need to complete both steps.
So I suggest something like: `Step 2: Database setup`. This section could then contain an introduction stating that only one is needed followed by subsections for MariaDB as well as PostgreSQL.

Reply:
it was triggered by the `This step` x) I'll change it to make it more clear